specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/storage.json

```diff
@@ -4448,6 +4713,128 @@
     },
+    "LocalUsers": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalUser"
+          },
+          "description": "The local users associated with the storage account."
+        }
+      },
+      "description": "List storage account local users."
+    },
+    "LocalUser": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LocalUserProperties",
+          "x-ms-client-flatten": true,
+          "description": "Storage account local user properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The local user associated with the storage accounts."
+    },
+    "LocalUserProperties": {
+      "properties": {
+        "permissionScopes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PermissionScope"
+          },
+          "description": "The permission scopes of the local user."
+        },
+        "homeDirectory": {
+          "type": "string",
+          "description": "Optional, local user home directory."
+        },
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique Security Identifier that is generated by the server."
+        },
+        "sharedKey": {
+          "$ref": "#/definitions/SharedKey"
+        },
+        "sshPassword": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Auto generated password by the server for SSH authentication if hasSshPassword is set to true on the creation of local user."
+        },
+        "hasSharedKey": {
+          "type": "boolean",
+          "description": "Indicates whether shared key exists. Set it to false to remove existing shared key."
+        },
+        "hasSshKey": {
+          "type": "boolean",
+          "description": "Indicates whether ssh key exists. Set it to false to remove existing SSH key."
+        },
+        "hasSshPassword": {
+          "type": "boolean",
+          "description": "Indicates whether ssh password exists. Set it to false to remove existing SSH password."
+        }
+      },
+      "description": "The Storage Account Local User properties."
+    },
+    "LocalUserKeys": {
+      "properties": {
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sharedKey": {
+          "$ref": "#/definitions/SharedKey"
+        }
+      },
+      "description": "The Storage Account Local User keys."
+    },
+    "PermissionScope": {
+      "properties": {
```

Comment:
Could you specify which properties of the permission scope is optional? Or they are all required?

Reply:
they are all required. Updated the definition.